File: Python/1045.py
import math

A, B, C = map(float, input().split())
a = A
b = B
c = C

if B > A:
    A = b
    B = a

if C > A:
    C = A
    A = c

if A >= (B + C):
    print("NAO FORMA TRIANGULO")
else:
    if math.pow(A, 2) == math.pow(B, 2) + math.pow(C, 2):
        print("TRIANGULO RETANGULO")
    elif math.pow(A, 2) > math.pow(B, 2) + math.pow(C, 2):
        print("TRIANGULO OBTUSANGULO")
    elif math.pow(A, 2) < math.pow(B, 2) + math.pow(C, 2):
        print("TRIANGULO ACUTANGULO")

    if A == B and B == C:
        print("TRIANGULO EQUILATERO")
    elif A == B or A == C or B == C:
        print("TRIANGULO ISOSCELES")

File: C++/1045.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {

    double A,B,C,a,b,c;
    cin >> A >> B >> C;
    a=A;
    b=B;
    c=C;
    if(B>A) {
        A=b;
        B=a;

    }
    if(C>A) {
        C=A;
        A=c;

    }
    if(A>= (B+C)) {
        cout << "NAO FORMA TRIANGULO" << endl;
    } else {
        if(pow(A,2) == pow(B,2)+ pow(C,2)) {
            cout << "TRIANGULO RETANGULO" <<  endl;

        } else if(pow(A,2) > pow(B,2)+ pow(C,2)) {
            cout << "TRIANGULO OBTUSANGULO" << endl;
        } else if(pow(A,2) < pow(B,2)+pow(C,2)) {
            cout << "TRIANGULO ACUTANGULO" << endl;
        }

        if(A==B && B==C) {
            cout << "TRIANGULO EQUILATERO" << endl;
        } else if(A==B  ||A==C || B==C) {
            cout << "TRIANGULO ISOSCELES" << endl;
        }

    }








return 0;
}
